### PR TITLE
Enable SSO for XDMoD Docker

### PR DESCRIPTION
**Pull request by plessbd**
This "works" but has an unrelated issue: 'Could not execute: /usr/sbin/sendmail -t -i' that will be resolved with getting sendmail to "work" with xdmod.  However I think since it is not related to SSO it does not belong in this PR

### CODE REVIEW

File: xdmod/entrypoint.sh [hunk 1]
```diff
@@ -40,8 +43,8 @@ then
         xdmod-ingestor -v
     fi
 
-    echo "---> Starting sshd on xdmod..."
-    /usr/sbin/sshd -e
+    echo "---> Setup XDMoD SSO..."
+    /srv/xdmod/scripts/xdmod-setup-sso.sh
```
Comment (aebruno) [inline xdmod/entrypoint.sh:47]: Does this need to execute every time the container is started? If not, perhaps we move this to the initialization part of the entrypoint?

**Comment by aebruno** [comment]
Can we skin this? At first I thought I was at a completely different site trying to phish my password?

xdmod-saml-login

**Comment by plessbd** [comment]
Can we, yes it is possible: https://simplesamlphp.org/docs/1.15/simplesamlphp-theming
However, not a priority in the time we have.

**Comment by aebruno** [comment]
> This "works" but has an unrelated issue: 'Could not execute: /usr/sbin/sendmail -t -i' that will be resolved with getting sendmail to "work" with xdmod. However I think since it is not related to SSO it does not belong in this PR

To test, I just did a `yum install postfix` in the xdmod container and was able to login. However, logout doesn't appear to work. If I login as hpcadmin, then click logout, then click login. I'm re-logged in as the hpcadmin user with no option to login as another user. This might make it difficult to showcase other users during the tutorial? Is there an easy fix for this without clearing cookies?

**Comment by plessbd** [comment]
Currently, no a logout of the SP does not trigger a logout from the IdP so a clearing of cookies will need to be done to change the user.

**Comment by aebruno** [comment]
> I just did a `yum install postfix` in the xdmod container and was able to login. 

Want to just add installing postfix to this PR? Seems to make xdmod happy and postfix server is not running so no email should be sent?

**Comment by plessbd** [comment]
I added #13 to have it just send the emails to a file so that if we want to show them we could without having to get into postfix

**Pull request merged**